src/imio/events/core/indexers.py

```
@@ -68,7 +68,10 @@ def get_local_category(obj, lang):
         IVocabularyFactory, "imio.events.vocabulary.EventsLocalCategories"
     )
     vocabulary = factory(obj, lang=lang)
-    term = vocabulary.getTerm(obj.local_category)
+    try:
+        term = vocabulary.getTerm(obj.local_category)
+    except LookupError:
+        raise AttributeError
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)